### PR TITLE
User Likes closes #38 closes #69 closes #101

### LINKED ISSUE #38: set up database for caching and serving festival data

### LINKED ISSUE #69: save spotify account id in db

### LINKED ISSUE #101: allow user to save favourite festivals

### PR DESCRIPTION
**Pull request by JamesAdamHughes**
- add fucking database son
- 3 models, Users, Events, UserEventLikes
- Save event id in db when first running server
- Save user id in db when user first logs in with spotify (this is also stored in session cookie)
- User can like event, creates entry in UserEventLikes that links the two ids
  - When event detail retrieved again, checks if the user has liked it 
  - Returns to the client if the user has liked it or not

### CODE REVIEW

File: public/app/services/SpotifyService.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ angular.module('festerrApp').factory('SpotifyService', function($q, $location, $
                     method: 'get',
                     credentials: 'include'
                 }).then(function(res) {
-                    console.log("Got " + res.artists.length + " Spotify Artists %o", res.artists);
+                    // console.log("Got " + res.artists.length + " Spotify Artists %o", res.artists);
```
Comment (louisditzel) [inline public/app/services/SpotifyService.js]: If we don't need this, then remove it 😉

File: server.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 var request = require('request');
 var fs = require('fs');
-var secretString = "9t6aHMrAauERxkR";
+var models = require("./models"); // Database models
 var sessions = require("client-sessions");
 
 var app = express();
 
+// Secret string for sessions cookie
+var secretString = "9t6aHMrAauERxkR";
```
Comment (louisditzel) [inline server.js]: Maybe add this to the config/api_keys.json file, or at least in its own config file

**Comment by JamesAdamHughes** [comment]
- Force spotify refresh on every page load to ensure session cookie always contains the userID if one exists
- Remove unneeded expanding code

**Comment by JamesAdamHughes** [comment]
Add this line to setEnvVars.js

process.env.session_secret = "9t6aHMrAauERxkR";

**Pull request merged**